examples/credential-registry/src/lib.rs

```diff
@@ -331,7 +337,7 @@ impl<S: HasStateApi> State<S> {
     }
 
     fn remove_revocation_key(&mut self, pk: PublicKeyEd25519) -> ContractResult<()> {
-        self.issuer_keys.remove(&pk);
+        ensure!(self.issuer_keys.remove(&pk), ContractError::KeyDoesNotExist);
```

Comment:
ensure!(self.revocation_keys.remove(&pk), ContractError::KeyDoesNotExist);

Reply:
Yes, did not notice that it was incorrect already :)